pydantic/main.py

```diff
@@ -303,15 +303,16 @@ def dict(
         include: Union['SetIntStr', 'DictIntStrAny'] = None,
         exclude: Union['SetIntStr', 'DictIntStrAny'] = None,
         by_alias: bool = False,
-        skip_defaults: bool = False,
+        exclude_unset: bool = False,
+        exclude_defaults: bool = False,
```

Comment:
I changed the default value to None so I can tell if it got set -- that way, you get a warning if you were specifying it as True *or* False (since, if you were specifying as `False`, it would start erroring if/when we removed it.) I think a more robust approach might be to pick up `**kwargs`, but that feels like overkill (and might be computationally expensive too).

I would imagine there are some neat decorators that could be used to perform this, but I'm also fine with the approach implemented here, or with leaving everything defaulting to False and just not warning people if they are passing `skip_defaults=False` somewhere. I leave it up to you